Description: Allow building with Debian's pydata-sphinx-theme

This is currently too old to have all the options upstream expects

--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -245,16 +245,7 @@ html_theme_options = {
     "footer_start": ["pandas_footer", "sphinx-version"],
     "github_url": "https://github.com/pandas-dev/pandas",
     "twitter_url": "https://twitter.com/pandas_dev",
-    "logo": {"image_dark": "https://pandas.pydata.org/static/img/pandas_white.svg"},
-    "navbar_align": "left",
-    "navbar_end": ["version-switcher", "theme-switcher", "navbar-icon-links"],
-    "switcher": {
-        "json_url": "https://pandas.pydata.org/versions.json",
-        "version_match": switcher_version,
-    },
-    # This shows a warning for patch releases since the
-    # patch version doesn't compare as equal (e.g. 2.2.1 != 2.2.0 but it should be)
-    "show_version_warning_banner": False,
+    "navbar_end": ["navbar-icon-links"],
     "icon_links": [
         {
             "name": "Mastodon",
--- a/doc/_templates/sidebar-nav-bs.html
+++ b/doc/_templates/sidebar-nav-bs.html
@@ -1,9 +1,9 @@
 <nav class="bd-links" id="bd-docs-nav" aria-label="Main navigation">
   <div class="bd-toc-item navbar-nav">
     {% if pagename.startswith("reference") %}
-    {{ generate_toctree_html("sidebar", maxdepth=4, collapse=True, includehidden=True, titles_only=True) }}
+    {{ generate_nav_html("sidebar", maxdepth=4, collapse=True, includehidden=True, titles_only=True) }}
     {% else %}
-    {{ generate_toctree_html("sidebar", maxdepth=4, collapse=False, includehidden=True, titles_only=True) }}
+    {{ generate_nav_html("sidebar", maxdepth=4, collapse=False, includehidden=True, titles_only=True) }}
     {% endif %}
   </div>
 </nav>
